.github/workflows/test.yaml

```diff
@@ -40,6 +40,27 @@ jobs:
           west-group-filter: -tools,-bootloader,-babblesim
           west-project-filter: -nrf_hw_models
 
+      - name: Set a timestamp variable
+        shell: bash
+        run: |
+          echo "CCACHE_TIMESTAMP=$(date +%s)" >> $GITHUB_ENV
```

Comment:
this could probably go in the other file, together with `CCACHE_IGNOREOPTIONS` and the others

Reply:
Right, pushed a WIP commit moving stuff around a bit.